recipes/mystran/build.sh

```diff
@@ -0,0 +1,7 @@
+cmake -B build -S . \
+      -D CMAKE_BUILD_TYPE="Release" \
+      -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX
+
+make -C build install
+mkdir -p $PREFIX/bin
```

Comment:
Any make check or ctest that we can run here?

Reply:
not sure if there is much testing available. I added some test files which run with mystran.

Reply:
If you believe what you have here good enough to test it, we can merge it.

Reply:
Yes, maybe there will be more tests avaoilable in the future